recipe/patches/0001-Use-conda-sysroot-when-building-recipes.patch

```diff
@@ -0,0 +1,21 @@
+--- pyqtbuild/bindings.py	2021-10-28 04:53:13.000000000 -0500
++++ pyqtbuild/bindings.py	2022-02-17 11:09:20.141023450 -0500
+@@ -146,6 +146,18 @@
+         buildable.include_dirs.extend(self.include_dirs)
```

Comment:
Where are these patches from? Assuming you wrote them (and have a branch in a local clone of pyqt), can you please do `git format-patch <tag_name_of_6.8.1> --no-signature`? This will provide authorship attribution, and provides an opportunity to provide an explanatory commit message.

Reply:
This is just for testing purposes--this won't be in the PR; this is going to be applied upstream.